pkg/porter/mixins.go

```diff
@@ -64,6 +66,49 @@ func (p *Porter) ListMixins() ([]mixin.Metadata, error) {
 	return mixins, nil
 }
 
+// SearchMixins searches the internal remote mixins list according to the provided options
+func (p *Porter) SearchMixins(opts mixin.SearchOptions) error {
+	box := packr.New("get.porter.sh/porter/pkg/mixin/directory", "../mixin/directory")
+	mixinSearcher := mixin.NewSearcher(box)
+
+	mixinList, err := mixinSearcher.Search(opts)
+	if err != nil {
+		return err
+	}
+	return p.PrintPackages(opts, mixinList)
+}
+
+// PrintPackages prints the provided package list according to the provided options
```

Comment:
This should move to the `pkgmgmt` package, but I was thinking we'd do so in a future refactor PR when we intro a plugin directory and searching.  WYDT?

Reply:
If we follow our current structure (printing logic inside of the porter pkg), then SearchMixins would be renamed to SearchPackages, NewSearcher would move to pkgmgmt, but SearchPackages and PrintPackages would stay in pkg/porter. Yeah this can all happen in a folllow up